src/screens/CandidateDetail/CandidateDetail.js

```diff
@@ -5,10 +5,13 @@ import { Avatar, Icon } from 'react-native-elements';
 
 const CandidateDetail = props => (
   <View style={styles.container}>
-    <CandidateSummary
-      {...props.summary}
-      onToggleFavorite={props.toggleFavorite}
-    />
+    {props.summary
+      ? <CandidateSummary
+        {...props.summary}
+        onToggleFavorite={props.toggleFavorite}
+      />
+      : <Text>Loading...</Text>
```

Comment:
Thanks for thinking about loading states. I think we should reach out to the design team and see if they have an idea for what would look best in the app